force-app/main/default/lwc/attendance/__tests__/attendance.test.js

```diff
@@ -21,11 +22,13 @@ const mockGenerateRoster = require("./data/generateRoster.json");
 const mockGenerateRosterEmpty = require("../../__tests__/data/emptyList.json");
 const mockGetFieldSet = require("./data/getFieldSet.json");
 const mockWiredSession = require("./data/wiredSession.json");
+const mockWiredPermissions = require("./data/wiredPermissions.json");
```

Comment:
What about adding a couple new tests with the empty state and when perms return false?